npc/re/merchants/Dealer_Update.txt

```diff
@@ -0,0 +1,182 @@
+		movenpc "Trading Merchant#iz", 72,110,3;
+	
+		//= Tool Dealer Before Upgrade!
+		unloadnpc "Tool Dealer#alb";
+		unloadnpc "Tool Dealer#alb2";
+		unloadnpc "Tool Dealer#alde";
+		unloadnpc "Tool Dealer#ama";
+		unloadnpc "Tool Dealer#ayo";
+		unloadnpc "Tool Dealer#bra";
+		unloadnpc "Tool Dealer#cmd";
+		unloadnpc "Tool Dealer#cmd2";
+		unloadnpc "Tool Dealer#ein";
+		unloadnpc "Tool Dealer#gef";
+		unloadnpc "Tool Dealer#gef2";
+		unloadnpc "Tool Dealer#gon";
+		unloadnpc "Tool Dealer#hu";
+		unloadnpc "Tool Dealer#hu2";
+		unloadnpc "Tool Dealer#yuno";
+		unloadnpc "Tool Dealer#yuno1";
+		unloadnpc "Tool Dealer#yuno2";
+		unloadnpc "Tool Dealer#lhz";
+		unloadnpc "Tool Dealer#lhz2";
+		unloadnpc "Tool Dealer#lou";
+		unloadnpc "Tool Dealer#xmas";
+		unloadnpc "Tool Dealer#mosk";
+		unloadnpc "Tool Dealer#pay";
+		unloadnpc "Tool Dealer#pay2";
+		unloadnpc "Tool Dealer#prt";
+		unloadnpc "Tool Dealer#prt1";
+		unloadnpc "Tool Dealer#ra";
+		unloadnpc "Tool Dealer#tu";
+		unloadnpc "Tool Dealer#um";
+		unloadnpc "Tool Dealer#ve";
+		unloadnpc "Tool Dealer#moc1";
+		unloadnpc "Tool Dealer#moc2";
+		unloadnpc "Tool Dealer#moc3";
+		unloadnpc "Tool Dealer#moc4";
+	
+	} else {
+	
+		//= About Sellers
+		unloadnpc "#over_arrow";
+		unloadnpc "#over_bullet";
+		unloadnpc "#over_kunai";
+		unloadnpc "#over_madogear";
+		unloadnpc "#over_talisman";
+		
+		//= Tool Dealer After Upgrade!
+		unloadnpc "Tool Dealer#Updade_Alb";
+		unloadnpc "Tool Dealer#Updade_Alb2";
+		unloadnpc "Tool Dealer#Updade_Alde";
+		unloadnpc "Tool Dealer#Updade_Ama";
+		unloadnpc "Tool Dealer#Updade_Ayo";
+		unloadnpc "Tool Dealer#Updade_Bra";
+		unloadnpc "Tool Dealer#Updade_Cmd";
+		unloadnpc "Tool Dealer#Updade_Cmd2";
+		unloadnpc "Tool Dealer#Updade_Ein";
+		unloadnpc "Tool Dealer#Updade_Gef";
+		unloadnpc "Tool Dealer#Updade_Gef2";
+		unloadnpc "Tool Dealer#Updade_Gon";
+		unloadnpc "Tool Dealer#Updade_Hu2";
+		unloadnpc "Tool Dealer#Updade_Yuno";
+		unloadnpc "Tool Dealer#Updade_Yuno1";
+		unloadnpc "Tool Dealer#Updade_Yuno2";
+		unloadnpc "Tool Dealer#Updade_Lhz";
+		unloadnpc "Tool Dealer#Updade_Lhz2";
+		unloadnpc "Tool Dealer#Updade_Lou";
+		unloadnpc "Tool Dealer#Updade_Xmas";
+		unloadnpc "Tool Dealer#Updade_Mosk";
+		unloadnpc "Tool Dealer#Updade_Pay";
+		unloadnpc "Tool Dealer#Updade_Pay2";
+		unloadnpc "Tool Dealer#Updade_Prt";
+		unloadnpc "Tool Dealer#Updade_Prt1";
+		unloadnpc "Tool Dealer#Updade_Ra";
+		unloadnpc "Tool Dealer#Updade_Tu";
+		unloadnpc "Tool Dealer#Updade_Um";
+		unloadnpc "Tool Dealer#Updade_Ve";
+		unloadnpc "Tool Dealer#Updade_Moc1";
+		unloadnpc "Tool Dealer#Updade_Moc2";
+		unloadnpc "Tool Dealer#Updade_Moc3";
+		unloadnpc "Tool Dealer#Updade_Moc4";
+	
+	}
+	end;
+}
+
+//= Tool Dealers
+//============================================================
+alb2trea,87,65,5	shop	Tool Dealer#Updade_Alb	83,611:-1,602:-1,601:-1,23280:-1,23288:-1,645:-1,656:-1,657:-1,501:-1,502:-1,503:-1,504:-1,506:-1,610:-1,713:-1,717:-1,1771:-1
```

Comment:
Can you replace "Updade" with "Update" here?
I dont want to suggest it via Github since you might call it from some other places.